back/spec/services/local_project_copy_service_spec.rb

```diff
@@ -253,13 +251,18 @@
     end
 
     it 'copies basic phase attributes' do
+      ignore_attributes = %i[
+        id project_id start_at end_at updated_at created_at
+        voting_method voting_max_total voting_min_total voting_max_votes_per_idea voting_term
+        survey_embed_url survey_service
```

Comment:
See you've added the survey attributes here, but guess they weren't causing any issues before?

Reply:
No, they were not causing issues. However, the voting_ and survey_ attributes are not included in the copy template when the participation context is not respectively voting or survey. Therefore, if in the test the survey_embed_url would be set for a non-survey context, the test would fail (because the value is changed to nil).